.pre-commit-config.yaml

```diff
@@ -8,18 +8,64 @@ repos:
     rev: 22.3.0
     hooks:
       - id: black
+        name: (BF) black
         args: [--preview]
-        files: ^backend/
+        files: ^backend/benefit
+      - id: black
+        name: (KS) black
+        args: [--preview]
+        files: ^backend/kesaseteli
+      - id: black
+        name: (TET) black
+        args: [--preview]
+        files: ^backend/tet
+      - id: black
+        name: (Shared) black
+        args: [--preview]
+        files: ^backend/shared
   - repo: https://github.com/pycqa/isort
     rev: 5.12.0
```

Comment:
Benefit/Kesaseteli/TET backends use an older isort 5.10.1 at the moment according to their requirements-dev.txt files. Maybe black/isort/flake8 versions could be updated to their newest versions in this file and in all the backends' requirements-dev.txt files to make them be in sync. At the least the versions should be the same for all IMHO.

Reply:
Versions should be in sync now. Also added the pre-commit dependency to kesaseteli/tet also.